google/cloud/vision/geometry.py

```diff
@@ -34,10 +34,8 @@ def from_api_repr(cls, response_vertices):
         :rtype: :class:`~google.cloud.vision.geometry.BoundsBase`
         :returns: Instance of BoundsBase with populated verticies.
         """
-        vertices = []
-        for vertex in response_vertices['vertices']:
-            vertices.append(Vertex(vertex.get('x', None),
-                                   vertex.get('y', None)))
+        vertices = [Vertex(vertex.get('x', None), vertex.get('y', None)) for
+                    vertex in response_vertices.get('vertices', [])]
```

Comment:
Here, `vertices` is naturally a list, so having an empty one (where the back-end didn't populate it) feels right.